frames/src/main/java/com/checkout/frames/mapper/BillingFormAddressToBillingAddressMapper.kt

```diff
@@ -0,0 +1,12 @@
+package com.checkout.frames.mapper
+
+import com.checkout.base.mapper.Mapper
+import com.checkout.frames.screen.billingaddress.billingaddressdetails.models.BillingAddress
+import com.checkout.frames.screen.paymentform.model.BillingFormAddress
+
+internal class BillingFormAddressToBillingAddressMapper : Mapper<BillingFormAddress?, BillingAddress> {
```

Comment:
nit: we can make a companion object containing a lazy instance so the manager can just take the INSTANCE, for example:

```kotlin
internal class BillingFormAddressToBillingAddressMapper : Mapper<BillingFormAddress?, BillingAddress> {

    override fun map(from: BillingFormAddress?) =
        BillingAddress(
        name = from?.name, phone = from?.phone, address = from?.address ?: BillingAddress().address
    )

    internal companion object {
        internal val INSTANCE by lazy { BillingFormAddressToBillingAddressMapper() }
    }
}

// then in the PaymentModule we can do this 
fun paymentStateManager(
    supportedCardSchemes: List<CardScheme>,
    prefillData: PrefillData?
): PaymentStateManager =
    PaymentFormStateManager(supportedCardSchemes, prefillData, BillingFormAddressToBillingAddressMapper.INSTANCE)
```